homeassistant/components/lidarr/sensor.py

```diff
@@ -41,7 +42,7 @@ def get_modified_description(
 ) -> tuple[LidarrSensorEntityDescription[T], str]:
     """Return modified description and folder name."""
     desc = deepcopy(description)
-    name = mount.path.rsplit("/")[-1].rsplit("\\")[-1]
+    name = os.path.basename(os.path.normpath(mount.path)).lower()
```

Comment:
Is normpath needed here?  I don't think it is necessary if you only want the basename.

Reply:
removing normpath leaves a trailing underscore in the `unique_id` in some cases. So that is not ideal.